packages/perseus/src/widgets/interactive-graphs/graphs/use-draggable.test.tsx

```diff
@@ -216,7 +216,7 @@ describe("useDraggable", () => {
         expect(screen.getByRole("button")).toHaveFocus();
 
         // Act:
-        await userEvent.keyboard("{arrowright>1}{arrowup>1}");
```

Comment:
The `>1` construct within curly braces `{}` causes the `userEvent` library to not lift the key. So this test is pressing the right arrow, holding it down, and pressing the up arrow. It ends with both keys down. 

See: https://testing-library.com/docs/user-event/keyboard/